includes/processors/class.llms.processors.php

```diff
@@ -103,7 +103,7 @@ private function load_all() {
 		 *
 		 * @param string[] $classes A list of processor class ids/slugs.
 		 */
-		$classes = apply_filters( 'llms_load_processors', $this->classes );
+		$this->classes = apply_filters( 'llms_load_processors', $this->classes );
```

Comment:
```suggestion
		$classes = apply_filters( 'llms_load_processors', $this->classes );
```
What about  and cycle over `$classes` in the foreach below?
My brain likes it more, b/c a member is not altered by a filter in a method that is not supposed to do it, I guess...
What do you think?